.github/workflows/test.yml

```diff
@@ -32,16 +34,23 @@ jobs:
         run: |
           CUDA_TAG=cpu pip install -r requirements.torch.txt --find-links https://download.pytorch.org/whl/torch_stable.html
           pip install -r requirements.test.txt
```

Comment:
Oops, I thought we had removed this `requirements.test.txt` since dev includes it.

Reply:
I also noticed it after and change it to `dev`.